torch/csrc/jit/ir.cpp

```diff
@@ -258,6 +259,12 @@ std::ostream& Node::print(
       out << " # " << filename << ":" << line << ":" << col;
     }
   }
+  if (kind() == prim::Constant &&
+      output()->type()->kind() == FunctionType::Kind) {
+    auto funcname =
+        output()->type()->expect<FunctionType>()->function()->name();
+    out << " # " << funcname;
```

Comment:
what happens if we're also printing source location? Doesn't this end up looking weird?

Reply:
Yeah, we might get double comments. I traded off for code simplicity here, but I can change it if you think it's bad :)

Reply:
Hm, maybe we should just make the function name an attribute of the node, for convenience? Similar to what we do with methods

Reply:
Your call, both seem fine to me

Reply:
> Hm, maybe we should just make the function name an attribute of the node, for convenience? Similar to what we do with CallMethod

That's a good idea, I'll try this version.

Reply:
Please take a look at #24446!